### PR TITLE
Mercure demo

### PR DESCRIPTION
**Pull request by dunglas**
Add Mercure support in the demo (WIP).

See: https://github.com/api-platform/core/pull/2282 and https://dunglas.fr/2018/10/http-2-speed-up-your-apps-and-dispatch-real-time-updates-symfony-and-api-platforms-features-announcement/

### CODE REVIEW

File: client/src/actions/book/create.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+export function loading(loading) {
+  return { type: 'BOOK_CREATE_LOADING', loading };
+}
+
+export function success(created) {
+  return { type: 'BOOK_CREATE_SUCCESS', created };
+}
+
+export function create(values) {
```
[the remark this answers is not in the document]
Comment (dunglas) [reply client/src/actions/book/create.js]: As this is generated code, we try to use as few libraries as possible (especially not one having only 5 contributors).

File: client/src/actions/book/create.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { SubmissionError } from 'redux-form';
+import fetch from '../../utils/fetch';
+
+export function error(error) {
+  return { type: 'BOOK_CREATE_ERROR', error };
```
[the remark this answers is not in the document]
Comment (dunglas) [reply client/src/actions/book/create.js]: Is it really interesting here? It adds boilerplate code for just 1 reuse.

File: client/src/actions/book/create.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (toofff) [reply client/src/actions/book/create.js]: I did not see that you used the API platform generator :(

**Comment by toofff** [comment]
@dunglas rebase PR please ;)

**Comment by dunglas** [comment]
Rebased. I removed the theme because it makes it very difficult to regenerate the client.

**Pull request merged**